.travis.yml

```diff
@@ -55,7 +55,7 @@ matrix:
         - mvn clean test jacoco:report coveralls:report
 
     - language: node_js
-      node_js: 6
+      node_js: lts/carbon
```

Comment:
Unify the travis test version as LTS name version, which has been already used in Dockerfile (`FROM node:carbon`)